js-forms_checkbox-input/js/index.js

```diff
@@ -12,13 +13,33 @@ function showTosError() {
   tosError.removeAttribute("hidden");
 }
 
+function hideSuccess() {
+  success.setAttribute("hidden", "");
+}
+
+function showSuccess() {
+  success.removeAttribute("hidden");
+}
+
+hideTosError();
+hideSuccess();
+
 form.addEventListener("submit", (event) => {
   event.preventDefault();
 
   // --v-- write your code here --v--
 
+  if (tosCheckbox.checked) {
+    hideTosError();
+    alert("Form submitted");
+    showSuccess();
+  } else {
+    showTosError();
+  }
   // --^-- write your code here --^--
 
   // eslint-disable-next-line no-alert
-  alert("Form submitted");
+  // alert("Form submitted");
```

Comment:
Everything perfect, besides one thing. Here we're just playing around with the code and that's fine, but when you want to get hired, do not include comments in your commits. Clean it up, keep it nice and tidy :sparkles:

Reply:
Good tip. Thank you, Philip! :)

Reply:
Just changed!